src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Server/SmiEventSink_Default.netcore.cs

```diff
@@ -6,32 +6,17 @@
 
 namespace Microsoft.Data.SqlClient.Server
 {
-    internal class SmiEventSink_Default : SmiEventSink
+    internal partial class SmiEventSink_Default : SmiEventSink
     {
-        private SqlErrorCollection _errors;
-        private SqlErrorCollection _warnings;
-
-
         internal bool HasMessages
         {
             get
             {
-                {
-                    bool result = (null != _errors || null != _warnings);
-                    return result;
-                }
+                bool result = (null != _errors || null != _warnings);
+                return result;
             }
         }
 
-        virtual internal string ServerVersion
-        {
-            get
-            {
-                return null;
-            }
-        }
-
-
         protected virtual void DispatchMessages()
         {
             // virtual because we want a default implementation in the cases
```

Comment:
Avoid the code redundancy even just a function definition. This method's defined on both projects.